wallet/node/Main.hs

```diff
@@ -139,6 +139,7 @@ action (WalletNodeArgs (cArgs@CommonNodeArgs{..}) (wArgs@WalletArgs{..})) =
     conf :: ConfigurationOptions
     conf = CLI.configurationOptions $ CLI.commonArgs cArgs
 
+
```

Comment:
Are you serial? Why do you commit it every time? Also `stack clean cardano-sl-wallet` works fine.